examples/simpleapp/mc_sc_workflow_example.md

```diff
@@ -176,7 +194,7 @@ As a result id of the transaction and id of the sidechain will be printed:
 ```
 
 Parameters for `sc_create` are:
-1. **"version"**: the version of the sidechain. Versions `0`, `1` and `2` are allowed by Zendoo. Version `1` is recommended to be used.
+1. **"version"**: the version of the sidechain. Versions `0`, `1` and `2` are allowed by Zendoo. Version `1` or `2` is recommended to be used.
```

Comment:
Only version 2 is recommended to use. Version 1 is deprecated, and is used only by legacy sidechains

Reply:
Version1 is not deprecated yet.